test/e2e/tests/privacy/polling.spec.ts

```diff
@@ -0,0 +1,486 @@
+          const ethCallInfuraRequests = infuraJsonRpcRequests.filter(
+            (obj) =>
+              obj.method === 'eth_call' &&
+              (obj.params as unknown[])?.[1] === '3',
+          );
+
+          const ethGetBalanceInfuraRequests = infuraJsonRpcRequests.filter(
+            (obj) =>
+              obj.method === 'eth_getBalance' &&
+              (obj.params as unknown[])?.[1] === '3',
+          );
+
+          // We will call eth_getBalance for Sepolia and Linea Sepolia because multicall is not available for them
+          expect(ethGetBalanceInfuraRequests.length).toEqual(2);
+          // We will call eth_call for linea mainnet and mainnet
+          expect(ethCallInfuraRequests.length).toEqual(2);
+        } else {
+          expect(
+            infuraJsonRpcRequests.some(
+              (obj) => obj.method === 'eth_blockNumber',
+            ),
+          ).toBeTruthy();
+          expect(
+            infuraJsonRpcRequests.some(
+              (obj) => obj.method === 'eth_getBlockByNumber',
+            ),
+          ).toBeTruthy();
+          expect(
+            infuraJsonRpcRequests.some((obj) => obj.method === 'eth_call'),
+          ).toBeTruthy();
+        }
+      },
+    );
+  });
+});
+
+describe('Token Detection', function () {
+  async function mockAccountApiForPortfolioView(mockServer: Mockttp) {
+    return [
+      await mockServer
+        .forGet(
+          'https://accounts.api.cx.metamask.io/v2/accounts/0x5cfe73b6021e818b776b421b1c4db2474086a7e1/balances',
+        )
+        .withQuery({
+          networks: '1,59144',
+        })
+        .thenCallback(() => ({
+          statusCode: 200,
+          json: {
+            count: 0,
+            balances: [
+              {
+                object: 'token',
+                address: '0x0000000000000000000000000000000000000000',
+                symbol: 'ETH',
+                name: 'Ether',
+                type: 'native',
+                timestamp: '2015-07-30T03:26:13.000Z',
+                decimals: 18,
+                chainId: 1,
+                balance: '20',
+              },
+            ],
+            unprocessedNetworks: [],
+          },
+        })),
+    ];
+  }
+  it('should make calls to account api as expected', async function () {
+    if (process.env.PORTFOLIO_VIEW) {
+      await withFixtures(
+        {
+          fixtures: new FixtureBuilder()
+            .withNetworkControllerOnMainnet()
+            .withPreferencesControllerShowNativeTokenAsMainBalanceDisabled()
+            .build(),
+          ganacheOptions: defaultGanacheOptions,
+          title: this.test?.fullTitle(),
+          testSpecificMock: mockAccountApiForPortfolioView,
+        },
+        async ({ driver, mockedEndpoint: mockedEndpoints }) => {
+          await loginWithoutBalanceValidation(driver);
+          const homepage = new HomePage(driver);
+          await homepage.check_pageIsLoaded();
+          await driver.delay(20000);
```

Comment:
can you put the number to a const here and call it for example `LARGE_DELAY_MS` ( i'm not good with naming )